deploy/kubernetes/helm/templates/bookie.yaml

```
@@ -165,7 +165,7 @@ spec:
             path: /bookkeeper/data/ledgers
 {{- end }}
 
-{{- if or (eq .Values.platform "gke") (eq .Values.platform "minikube") }}
+{{- if .Values.formatDisks }}
```

Instead of `formatDisks`, let's use `.Values.bookkeeper.useVolumeClaimTemplate`
```suggestion
{{- if .Values.bookkeeper.useVolumeClaimTemplate }}
```